### ISSUE TITLE
Introduce Symlex

### ISSUE THREAD

**Issue by waghanza** [the-benchmarker/web-frameworks#2129]
Hi @lastzero,

Would you mind to add (or consent that we add) [symlex](https://github.com/symlex/symlex) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

**Comment by lastzero** [comment]
That'd be cool! I'm personally pretty busy developing https://github.com/photoprism/photoprism right now, so ideally you manage to do this on your own... of course I can answer questions and if you don't get it done, I can probably do it somehow :)

Especially the new release with roadrunner would be great to test... be aware that roadrunner itself also has a configuration. Benchmark results will depend on the number of workers for example (ideal value should be the number of logical processors). You also have the option to use PHP CLI or FPM, FPM might be better for benchmarks (but didn't really test it yet as it was always good enough on prod).

**Comment by lastzero** [comment]
Did you see our REST example app? https://github.com/symlex/rest-api

Simply rename `example` to `user` 👍

It's still based on nginx, probably very similar to what [phpbenchmarks.com](http://www.phpbenchmarks.com/en/benchmark/symlex/4.2) uses for benchmarking Symlex 4.2.

So the task for benchmarking Symlex 4.4.1 with your Specs would be to combine it's RoadRunner architecture with the app code of our REST example.

**Comment by waghanza** [comment]
Yes, I'll do this implementation soon, probably based on the above link :+1: 

Moreover, the purpose is not the same as `phpbenchmark` as tests will run (long time schedule) on cloud

**Comment by lastzero** [comment]
Interesting. Let me know when you're done and want a review, and of course when you have first results!
